src/components/Header/index.tsx

```diff
@@ -0,0 +1,93 @@
+import clsx from 'clsx'
+import { useRouter } from 'next/router'
+import { ArrowLeftIcon, ExitIcon, MoreVerticalIcon } from '@/components/Icons'
+import { PropsWithChildren } from 'react'
+
+export const buttonOptions = ['none', 'exit', 'more', 'back'] as const
+type HeaderButton = (typeof buttonOptions)[number]
```

Comment:
```suggestion
type HeaderButton = 'none' | 'exit' | 'more' | 'back'
```

혹시 이 부분을 유니온 타입으로 작성하지 않은 이유가 있을까요??
`buttonOptions`는 어디에서 추가로 사용될 수 있는 걸까요?

Reply:
아 요건 storybook 코드에서 radio 형식으로 선택할 수 있게끔하다가 그러려면 option들을 수동으로 입력해야해서 불편하더라구요.
그래서 옵션값들을 상수로 정의했었습니다!
<img width="755" alt="image" src="https://user-images.githubusercontent.com/49135797/218307938-c118f34d-01cb-4026-8891-c694e0262e97.png">

```ts
// Header > index.stories.tsx
export default {
  title: 'components/Header',
  components: Header,
  parameters: {
    layout: 'fullscreen',
  },
  argTypes: {
    leftButton: {
      options: buttonOptions,
      control: { type: 'radio' },
    },
    rightButton: {
      options: buttonOptions,
      control: { type: 'radio' },
    },
  },
} as ComponentMeta<typeof Header>
```

Reply:
아하...! 스토리북을 위한 것이었군요. 감사합니다 :D